modules/openy_gc_livechat/openy_gc_livechat.module

```diff
@@ -18,3 +18,13 @@ function openy_gc_livechat_theme() {
     ],
   ];
 }
+
+/**
+ * Implements hook_cron().
+ */
+function openy_gc_livechat_cron() {
+  // Delete chat messages older than 30 days.
+  \Drupal::database()->delete('openy_gc_livechat__chat_history')
```

Comment:
Do we really want to execute it every time during the corn run or is it better to limit and execute it every hour or once a day?